scripts/deploy_assisted_service.sh

```diff
@@ -15,6 +15,14 @@ export OCP_SERVICE_PORT=$(( 7000 + $NAMESPACE_INDEX ))
 export OPENSHIFT_INSTALL_RELEASE_IMAGE=${OPENSHIFT_INSTALL_RELEASE_IMAGE:-}
 export ENABLE_KUBE_API=${ENABLE_KUBE_API:-false}
 export ENABLE_KUBE_API_CMD="ENABLE_KUBE_API=${ENABLE_KUBE_API}"
+export OPENSHIFT_VERSIONS=${OPENSHIFT_VERSIONS:-"{}"}
+
+if [[ ! -z ${ENABLE_KUBE_API:-} && "${OPENSHIFT_VERSIONS}" == "{}" ]]; then
+    rhcos_image="https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/4.7/4.7.0/rhcos-4.7.0-x86_64-live.x86_64.iso"
```

Comment:
I would prefer not to maintain another OCP version data. This default should be taken from the defaults JSON in assisted-service.
https://github.com/openshift/assisted-service/blob/4a57d53684600660d7d577e2e74b2d7e112503fe/default_ocp_versions.json#L9-L16

You could use `jq` to get the entry with `default` equals to `true` and remove `release_image` fields